test/arrays/shapes/skyline.future

```diff
@@ -1,2 +1,2 @@
-bug: valgrind failure
-#11149
+bug: forall-over-forall results in empty array
+#11039
```

Comment:
11039 is certainly related but doesn't seem to describe the current behavior so seems more of a distraction. I'd consider closing that one and making a new one with content pasted from your PR message.

Reply:
I created #11254 for the new skyline failure mode. However, #11039 still describes the current behavior and needs to be resolved.